index.js

```diff
@@ -4,9 +4,11 @@ var registryUrl = require('registry-url');
 var rc = require('rc');
 
 module.exports = function (name, version) {
+	var isScoped = name.indexOf('/') > -1;
 	var scope = name.split('/')[0];
 	var url = registryUrl(scope) +
-		encodeURIComponent(name).replace(/^%40/, '@');
+		encodeURIComponent(name).replace(/^%40/, '@') +
+		(!isScoped && version ? '/' + version : '');
```

Comment:
@destroyerofbuilds If that is the case (which would be a very strange decision for a good API, I have to admit) then I guess the PR shouldn't be merged. Thanks for pointing it out. Do you maybe have some idea to get the real version number that would be installed by npm having a string like "~1.2.3" that would use the deprecated API that currently supports such version as part of the URL? Thanks.